File: whiteboarding/height_checker.py
heights = [1,1,4,2,1,3]

class Solution:
    def heightChecker(self, heights):
        sorts = sorted(heights)
        count = 0
        for x, y in zip(heights, sorts):
            if x != y:
                count += 1
        return count
        
s = Solution()
print(s.heightChecker(heights))

File: whiteboarding/height_checker.js
// https://leetcode.com/problems/height-checker/


let heights = [1,1,4,2,1,3]
let heightChecker = function(heights) {
    let count=0;
    let arr=heights.slice().sort((a,b)=> a-b);
    for(i=0; i<heights.length; i++){
        if(heights[i]!==arr[i]) count++;
    }
    return count;
};
console.log(heightChecker(heights));